<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorParametersWidget</class>
 <widget class="QWidget" name="SensorParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensor Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sensor Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max. Usable Distance:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Angle of View:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Gimballed:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="directionalityComboBox">
       <item>
        <property name="text">
         <string>Directional</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Omni-Directional</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="maxDistanceSpinBox">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="angleOfViewSpinBox">
       <property name="minimum">
        <double>2.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="value">
        <double>25.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="gimballedCheckBox">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="text">
        <string>Awesome UAV Sensor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/16x16/crystal_project/16x16/filesystems/folder_documents.png</normaloff>:/icons/16x16/crystal_project/16x16/filesystems/folder_documents.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/16x16/crystal_project/16x16/devices/3floppy_unmount.png</normaloff>:/icons/16x16/crystal_project/16x16/devices/3floppy_unmount.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/16x16/crystal_project/16x16/actions/reload.png</normaloff>:/icons/16x16/crystal_project/16x16/actions/reload.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/16x16/crystal_project/16x16/actions/cancel.png</normaloff>:/icons/16x16/crystal_project/16x16/actions/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/16x16/crystal_project/16x16/actions/apply.png</normaloff>:/icons/16x16/crystal_project/16x16/actions/apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
